obal/data/roles/diff_package/tasks/git_package_info.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: get spec file
+  find:
+    pattern: "*.spec"
+    path: "{{ inventory_dir }}/packages/{{ inventory_hostname }}"
+  register: spec_file
+  tags:
+    - always
+
+- name: "Set spec file path, scl options and releasers"
+  set_fact:
+    spec_file_path: "{{ spec_file.files[0].path }}"
+
+- name: 'Get latest package info from git'
+  command: >
+    rpmspec --query
+      --undefine=dist
+      --queryformat=%{name}-%{version}-%{release}
+      {{ '--define "scl_prefix ' + scl + '-"' if scl else '' }}
```

Comment:
This currently fails when `scl` is not defined at all. Should the `scl` var be defined as `''` in `defaults/main.yml`?

Reply:
Added a default for this.